doc/classes/RemoteTransform.xml

```diff
@@ -82,14 +88,19 @@
 	</methods>
 	<members>
 		<member name="remote_path" type="NodePath" setter="set_remote_node" getter="get_remote_node">
+			The [NodePath] of the mirrored [code]remote node[/code].
 		</member>
 		<member name="update_position" type="bool" setter="set_update_position" getter="get_update_position">
+			If [code]true[/code] the [code]remote node[/code]'s position is mirrored.
 		</member>
 		<member name="update_rotation" type="bool" setter="set_update_rotation" getter="get_update_rotation">
+			If [code]true[/code] the [code]remote node[/code]'s rotation is mirrored.
 		</member>
 		<member name="update_scale" type="bool" setter="set_update_scale" getter="get_update_scale">
+			If [code]true[/code] the [code]remote node[/code]'s scale is mirrored.
 		</member>
 		<member name="use_global_coordinates" type="bool" setter="set_use_global_coordinates" getter="get_use_global_coordinates">
+			If [code]true[/code] global coordinates are used. Otherwise local coordinates are used.
```

Comment:
- Otherwise -> If [code]false[/code]
- Add default value (I'd like to know if it defaults to global or local)